src/helpers/constants/ios.ts

```diff
@@ -35,16 +35,11 @@ export const CIO_APPDELEGATEDECLARATION_REGEX =
   /@implementation AppDelegate(.|\n)/;
 
 export const CIO_APPDELEGATEHEADER_REGEX =
-  /@interface AppDelegate : EXAppDelegateWrapper <RCTBridgeDelegate>/;
+  /(@interface AppDelegate\s:\sEXAppDelegateWrapper\s?)(<([^>]+)>)?/;
```

Comment:
```suggestion
  /(@interface AppDelegate\s*:\s*EXAppDelegateWrapper\s*)(<([^>]+)>)?/;
```

There might be more or less whitespace characters so this suggestion could help prevent errors against that.